components/viewController/viewController.js

```diff
@@ -3,22 +3,23 @@ app1.controller('viewController', function ($scope) {
 
 	$scope.views = [
 		{
-			jTitle: 'Import class list',
+			jTitle: 'Import Class List',
 			jDescription: 'Import the names of your students by pasting in a list or entering them using the form.'
 		},
 		{
-			jTitle: 'Create template',
+			jTitle: 'Create Template',
 			jDescription: 'Create a template that you will use to quickly generate reports based on the criteria you set.'
```

Comment:
These are Jeremy's strings, and I feel that changing them is probably outside the scope of the PR (I changed capitalization of the titles since this was a visual update)